raiden/utils/__init__.py

```diff
@@ -26,6 +26,13 @@
 from raiden.utils.signing import sha3  # noqa
 
 
+class CanonicalIdentifier(NamedTuple):
```

Comment:
Should this really be in Utils? why not in `raiden/transfer`? similar to `queue_identifier`

Reply:
Yeah that's indeed a funny place. I would have expected to see this in typing or in raiden/transfer